models/DataObject/ClassDefinition/Service.php

```diff
@@ -358,4 +360,63 @@ public static function skipColumn($tableDefinitions, $table, $colName, $type, $d
 
         return false;
     }
+
+    /**
+     * @param array $implementsParts
+     * @param string $newInterfaces
+     * @return string
+     * @throws \Exception
+     */
+    public static function buildImplementsInterfaces($implementsParts = [], string $newInterfaces) {
```

Comment:
As it is not possible to just skip this parameter because the next one is required, the signature should be as follows:
```suggestion
    public static function buildImplementsInterfaces(array $implementsParts, string $newInterfaces) {
```